### ISSUE TITLE
Dependency error when Upgrading to Server 7.0.2 & Client 5.1.0

### ISSUE THREAD

**Issue by StefanKoenigMUC** [graphql-dotnet/graphql-client#456]
Hi,

I just upgraded from

Client 
`    <PackageReference Include="GraphQL.Client" Version="5.0.0" />
`to
`    <PackageReference Include="GraphQL.Client" Version="5.1.0" />
`
and the server from 
`    <PackageReference Include="GraphQL.Server.All" Version="6.1.0" />
`to 
`    <PackageReference Include="GraphQL.Server.All" Version="7.0.0" />
`
Since them I'm getting errors during dependency resolvement:

ISchema is not part of of an referenced assembly, only available in 5.0.0.

Is that a known issue when upgrading?

Thanks in advance

**Comment by sungam3r** [comment]
No. Could you please provide exact compiler/runtime error message?
